<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="ButtonGroup" name="btnGroup" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="MyFile" name="myfilePage"/>
      <widget class="QWidget" name="sharelistPage"/>
      <widget class="QWidget" name="rankingPage"/>
      <widget class="Transfer" name="transferPage"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonGroup</class>
   <extends>QWidget</extends>
   <header location="global">toolswt/buttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyFile</class>
   <extends>QWidget</extends>
   <header location="global">myfile.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Transfer</class>
   <extends>QWidget</extends>
   <header location="global">transfer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
